### PR TITLE
WordPress release for staged deployments

### PR DESCRIPTION
**Pull request by saisubodhMSFT**

### CODE REVIEW

File: WPRelease/CleanUp/cleanImageCache.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+steps:
+- script: |
+    docker ps -a
+    docker system prune -af
```
Comment (ZubaeyrMSFT) [inline WPRelease/CleanUp/cleanImageCache.yml:4]: Will this remove images from other builds or only WordPress image?

File: WPRelease/stage.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    pool: server
+    timeoutInMinutes: 4320 # job times out in 3 days
+    dependsOn: tagAndUpdateImages
+    condition: and(succeeded(), ne(${{ parameters.stageNumber }}, 7))
+    steps:
+    - task: ManualValidation@0
+      timeoutInMinutes: 1440 # task times out in 1 day
+      inputs:
+        notifyUsers: |
+            saisubodh@microsoft.com
```
Comment (ZubaeyrMSFT) [inline WPRelease/stage.yml:43]: Can you also add WordPress team email here?

File: WPRelease/stage.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+parameters:
+  stageNumber: 0
+  stageName: "Stage0"
```
Comment (ZubaeyrMSFT) [inline WPRelease/stage.yml:3]: Are these the default values? This be overwritten by the input parameters right?

File: WPRelease/stage.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        stageNumber: ${{ parameters.stageNumber }}
+        
+  - job: waitForValidation
+    displayName: Wait for external validation
+    pool: server
+    timeoutInMinutes: 4320 # job times out in 3 days
+    dependsOn: tagAndUpdateImages
+    condition: and(succeeded(), ne(${{ parameters.stageNumber }}, 7))
+    steps:
+    - task: ManualValidation@0
```
Comment (ZubaeyrMSFT) [inline WPRelease/stage.yml:39]: How does the manual validation work? I mean, I only see notification to user but no steps to take input from user.

**Review by ZubaeyrMSFT — CHANGES_REQUESTED** [review]

File: WPRelease/CleanUp/cleanImageCache.yml (hunk 1 above)
Comment (saisubodhMSFT) [reply WPRelease/CleanUp/cleanImageCache.yml:4]: It removes all images on the build agent.

File: WPRelease/stage.yml (hunk 3 above)
Comment (saisubodhMSFT) [reply WPRelease/stage.yml:3]: Correct.

File: WPRelease/stage.yml (hunk 4 above)
Comment (saisubodhMSFT) [reply WPRelease/stage.yml:39]: It waits for a user to either resume or reject. Currently, it rejects by default if no manual validation is received within 1 day.

**Review by ZubaeyrMSFT — APPROVED** [review]

**Pull request merged**